docs/manual/scala/guide/advanced/code/build-update-akka.sbt

```diff
@@ -0,0 +1,38 @@
+// #akka-update
+// The newer Akka version you want to use.
+val akkaVersion = "2.6.0-RC2"
+
+// Akka dependencies used by Lagom
+libraryDependencies ++= Seq(
```

Comment:
```suggestion
dependencyOverrides ++= Seq(
```
?

I'm never sure when `dependencyOverrides` is necessary and `libraryDependencies` is _just_ enough.

Reply:
`libraryDependencies` is "use this dependency with this version, but it can be evicted by a new version added transitively".

`dependencyOverrides` is "use this dependency with this version" even if a new version is required transitively.

In other words, you are right, we want `dependencyOverrides` since we are overriding the version used by Lagom. And it is possible that it is older than the version used by Lagom (meaning it will be evicted if we use `libraryDependencies`).